cyto_dl/nn/vits/seg.py

```diff
@@ -1,13 +1,14 @@
 from typing import List, Optional, Union
 
+import numpy as np
 import torch
 from einops.layers.torch import Rearrange
 from monai.networks.blocks import UnetOutBlock, UnetResBlock, UpSample
 
 from cyto_dl.nn.vits.mae import MAE_Encoder
 
 
-class SupperresDecoder(torch.nn.Module):
+class SuperresDecoder(torch.nn.Module):
```

Comment:
this is unrelated to crossMAE but just in this PR?

Reply:
oops, started as a general decoder update, I can rename the pr